packages/math-input/src/components/input/math-wrapper.ts

```diff
@@ -36,40 +33,15 @@ import {
     MathQuillUpdaterCallback,
 } from "./mathquill-types";
 
-function buildNormalFunctionCallback(command: string) {
-    return function (mathField: MathFieldInterface) {
-        mathField.write(`\\${command}\\left(\\right)`);
-        mathField.keystroke("Left");
-    };
-}
-
 const customKeyTranslator: Record<Key, MathQuillUpdaterCallback> = {
```

Comment:
So is `customKeyTranslator` the mobile instance of a key translator?  Could we name it as such (even in a future PR)?   Right now `custom` doesn't hit at why or what it's used for.